src/app_service/service/message/service.nim

```diff
@@ -1147,3 +1187,26 @@ proc resendChatMessage*(self: Service, messageId: string): string =
   except Exception as e:
     error "error: ", procName="resendChatMessage", errName = e.name, errDesription = e.msg
     return fmt"{e.name}: {e.msg}"
+
+# TODO: would be nice to make it async but in this case we will need to show come spinner in the UI during deleting the message
```

Comment:
Maybe we can add an issue to track it. It's worth doing this part of a refactor week in the future